Citavi 5 XML.js

```diff
@@ -0,0 +1,980 @@
+	"Book" : "book", //Buch (Monographie)
+	"BookEdited" : "book", //Buch (Sammelwerk)
+	"Broadcast" : "tvBroadcast", //Radio- oder Fernsehsendung
+	"CollectedWorks" : "book", //Schriften eines Autors
+	"ComputerProgram" : "computerProgram", //Software
+	"ConferenceProceedings" : "book", //Tagungsband
+	"Contribution" : "bookSection", //Beitrag in ...
+	"ContributionInLegalCommentary" : "bookSection", //Beitrag in Gesetzeskommentar
+	"CourtDecision" : "case", //Gerichtsentscheid
+	"File" : "manuscript", //Akte
+	"InternetDocument" : "webpage", //Internetdokument
+	"InterviewMaterial" : "interview", //Interviewmaterial
+	"JournalArticle" : "journalArticle", //Zeitschriftenaufsatz
+	"Lecture" : "presentation", //Vortrag
+	"LegalCommentary" : "book", //Gesetzeskommentar
+	"Manuscript" : "manuscript", //Manuskript
+	"Map" : "map", //Geographische Karte
+	"Movie" : "videoRecording", //Spielfilm
+	"MusicTrack" : "audioRecording", //Musiktitel in ...
+	"MusicAlbum" : "audioRecording", //Musikwerk / Musikalbum
+	"NewsAgencyReport" : "report", //Agenturmeldung
+	"NewspaperArticle" : "newspaperArticle", //Zeitungsartikel
+	"Patent" : "patent", //Patentschrift
+	"PersonalCommunication" : "email", //Persönliche Mitteilung
+	"PressRelease" : "report", //Pressemitteilung
+	"RadioPlay" : "podcast", //Hörspiel
+	"SpecialIssue" : "book", //Sonderheft, Beiheft
+	"Standard" : "report", //Norm
+	"StatuteOrRegulation" : "statute", //Gesetz / Verordnung
+	"Thesis" : "thesis", //Hochschulschrift
+	"Unknown" : "document", //Unklarer Dokumententyp
+	"UnpublishedWork" : "report" //Graue Literatur / Bericht / Report
+};
+
+
+function doImport() {
+	var doc = Zotero.getXML();
+	
+	//Groups will also be mapped to tags which can be assigned to
+	//items or notes.
+	var groups = ZU.xpath(doc, '//Groups/Group');
+	var rememberTags = {};
+	for (var i=0; i<groups.length; i++) {
+		var id = ZU.xpathText(groups[i], './@id');
+		var name = ZU.xpathText(groups[i], './Name');
+		var referenceGroups = ZU.xpath(doc, '//ReferenceGroups/OnetoN[contains(text(), "'+id+'")]|//KnowledgeItemGroups/OnetoN[contains(text(), "'+id+'")]');
+		for (var j=0; j<referenceGroups.length; j++) {
+			var refid = referenceGroups[j].textContent.split(';')[0];
+			if (rememberTags[refid]) {
+				rememberTags[refid].push(name);
+			} else {
+				rememberTags[refid] = [name];
+			}
+		}
+	}
+	
+	//Main information for each reference.
+	var item;
+	var references = ZU.xpath(doc, '//References/Reference');
+	var unfinishedReferences = [];
+	var itemIdList = {};
+	for (var i=0, n=references.length; i<n; i++) {
+		var type = ZU.xpathText(references[i], 'ReferenceType');
+		if (type && typeMapping[type]) {
+			item = new Zotero.Item(typeMapping[type]);
+		} else {
+			Z.debug("Not yet supported type: " + type);
+			Z.debug("Therefore use default type 'journalArticle'");
+			item = new Zotero.Item("journalArticle");
+		}
+		item.itemID = ZU.xpathText(references[i], './@id');
```

Comment:
what's the point of setting an item ID here? Zotero doesn't do anything with those. (I'll post these as single comments, since I'm not sure how quickly I'll be able to get through all of them).